.github/workflows/ci.yml

```diff
@@ -270,6 +270,39 @@ jobs:
 
           _make PLATFORM=virt
 
+  QEMUv7_check:
+    name: make check (QEMUv7)
+    runs-on: ubuntu-latest
+    container: jforissier/optee_os_ci:qemuv8_check2
+    steps:
+      - name: Remove /__t/*
+        run: rm -rf /__t/*
+      - name: Restore build cache
+        uses: actions/cache@v3
+        with:
+          path: /github/home/.cache/ccache
+          key: qemuv7_check-cache-${{ github.sha }}
+          restore-keys: |
+            qemuv7_check-cache-
+      - name: Checkout
+        uses: actions/checkout@v4
+      - shell: bash
+        run: |
+          # make check task
+          set -e -v
+          export LC_ALL=C
+          export BR2_CCACHE_DIR=/github/home/.cache/ccache
+          export CFG_TEE_CORE_LOG_LEVEL=0
+          WD=$(pwd)
+          cd ..
+          TOP=$(pwd)/optee_repo_qemu_v8
+          /root/get_optee_qemuv8.sh ${TOP}
```

Comment:
It looks strange to use a script named _get_optee_qemuv8.sh_ but if it makes sense, I'm fine.

Reply:
Oops! That's a big mistake. Will fix.

Reply:
Fixed